x-pack/test/functional/apps/infraops/home_page.ts

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
This applies to the other open PRs with tests as well: we should probably keep in sync with the plugin folder stucture and call the folder `infra` instead of `infraops` for now?

Reply:
yes, I'm always in favor of consistency :wink: If we want to agree on `infra` for now (possibly `ingest` after #24242), I can change it.

Reply:
:+1: I change mine too.